xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVAAPI.h

```diff
@@ -22,9 +22,16 @@
 
 #include "system.h"
 
+#ifdef HAS_GL
 #include "cores/VideoPlayer/VideoRenderers/LinuxRendererGL.h"
+using CRendererVAAPIBase = CLinuxRendererGL;
```

Comment:
so would you prefer to have `RendererVAAPIGL.h`, `RendererVAAPIGL.cpp` and `RendererVAAPIGLES.h`, `RendererVAAPIGLES.cpp` ???